manifests/restart.yaml

```diff
@@ -1,3 +1,73 @@
 format: yamanifest
 version: 1.0
---- {}
```

Comment:
I think you committed this by accident? This file should be unchanged?

Reply:
Yes, that's a mistake. I restored it